addon/components/frost-select.js

```diff
@@ -228,6 +230,24 @@ export default Component.extend({
 
     return `${selectedItems.length} items selected`
   },
+
+  @readOnly
+  @computed('width')
+
+  /**
+   * Get a style string based on the presence of some properties
+   * @param {String} width the width property specified
+   * @returns {String} the completed style string
+   */
+  style (width) {
+    let styles = ''
+
+    // if a property is not falsy, append it to the style string
+    // note that in the width case, we want the component interface to have absolute power over the width
+    // so it will override any max or win widths to ensure ultimate control
+    if (width) styles += `width: ${width}px; max-width: initial; min-width:initial; `
```

Comment:
I thought we had a linting rule against if-statements without blocks. Also, why are we appending to styles if we know styles is empty otherwise? This could just be an assignment.